### PR TITLE
Minor fixes for mock-iot-server

### PR DESCRIPTION
**Pull request by gordonwang0**
- Fix spacing and permissions on mock-iot-setup.sh
- Change mock-iot-server to use HashSet for Modules, since ordering isn't important.

### CODE REVIEW

File: identity/mock-iot-server/src/hub.rs [hunk 1]
```diff
@@ -3,7 +3,7 @@
 use crate::server::Response;
 
 fn list_modules(
-    modules: &std::collections::BTreeSet<aziot_identity_common::hub::Module>,
+    modules: &std::collections::HashSet<aziot_identity_common::hub::Module>,
```
Comment (nyanzebra) [inline identity/mock-iot-server/src/hub.rs:6]: This is for correctness or optimization?

Comment (gordonwang0) [reply identity/mock-iot-server/src/hub.rs:6]: Optimization. Ordering doesn't matter for this, so it's fine to use HashSet.

**Comment by arsing** [comment]
`Module` isn't `Eq + Hash` either, so this will still have the same problem when you eventually use the set.

**Comment by gordonwang0** [comment]
True, but since this is test code and we don't have tests that require it, I think it's fine as is. If we add tests in the future that require Modules to be inserted into the set, we can add the derives to the Module struct.

**Review by arsing — APPROVED** [review]

**Review by nyanzebra — APPROVED** [review]

**Pull request merged**